apps/transport/lib/transport_web/router.ex

```diff
@@ -349,7 +357,23 @@ defmodule TransportWeb.Router do
     |> Enum.any?(fn org -> org["slug"] == "equipe-transport-data-gouv-fr" end)
   end
 
-  defp transport_data_gouv_member(conn, _) do
+  # Check that a secret key is passed in the URL in the `export_key` query parameter
+  defp check_export_secret_key(%Plug.Conn{params: params} = conn, _) do
+    export_key_value = Map.get(params, "export_key", "")
+    expected_value = Application.fetch_env!(:transport, :export_secret_key)
+    key_matches = Plug.Crypto.secure_compare(export_key_value, expected_value)
+
+    if expected_value not in [nil, ""] and key_matches do
```

Comment:
`expected_value not in [nil, ""]` pour éviter qu'on donne accès à l'export CSV si jamais `Application.fetch_env!(:transport, :export_secret_key)` est nul/vide (peut arriver en local)